### ISSUE TITLE
Request option to match ALL words in a search

### ISSUE THREAD

**Issue by rhyddic** [daneren2005/Subsonic#865]
When I search for Simple Plan, I want to see everything in my collection by Simple Plan, no matter what album. I don't want to see Simple Minds or Robert Plant in the results. Nor would I like to see Ca Plane Pour Moi by Plastic Bertrand. Currently, if I search within quotes or add pluses before each word, I get the results I want, but logically, especially with large collections, match all words should be the default.

**Comment by KatyLB** [comment]
I agree, this is a problem at the moment. It's impossible to just put in part of a song title and an artist name and get the result for what you want, which makes finding a specific track harder than it needs to be.
